### ISSUE TITLE
How can we use this to train on our own dataset?

### ISSUE THREAD

**Issue by Kaushal28** [multimodallearning/pytorch-mask-rcnn#98]
The MaskRCNN implementation using TensorFlow has given samples here: https://github.com/matterport/Mask_RCNN/tree/master/samples

Is there similar samples available for this?

**Comment by dppselena** [comment]
@Kaushal28 Hello,have you solve this problem?

**Comment by Kaushal28** [comment]
@dppselena I used matterport's MaskRCNN: https://www.kaggle.com/kaushal2896/imaterialist-2020-starter-eda-mask-rcnn

**Comment by dppselena** [comment]
Thank u so much, I have implemented my own data set on this recurring model, and I am very happy to communicate.

发自我的iPhone
